src/player.cpp

```diff
@@ -11000,6 +11013,15 @@ bool player::wear_item( const item &to_wear, bool interactive )
         }
     }
 
+    // Check if we don't have both hands available before wearing a briefcase, shield, etc. Also occurs if we're already wearing one.
+    if (to_wear.has_flag("RESTRICT_HANDS") && !has_two_arms()) {
+        if(interactive) {
+            add_msg_if_player(m_info, _("You don't have a hand free to wear an %s")),
```

Comment:
That "an" could get weird in some cases. For example, "an bindle".
"The" or "that" would be safer.